### PR TITLE
production data from eia updated

### LINKED ISSUE #3016: Update nationwide production to CY 2017
I went to EIA's site and think that the 2017 nationwide production data is available. I know we are getting this from API, so I'm not sure if we just need to trigger a refresh or what.

Since we will have federal land data through 2017 with the next release. I think it would be good to have the nationwide through 2017 up there soon too.

### PR DESCRIPTION
**Pull request by master12**
Fixes #3016 

[:sunglasses: PREVIEW](https://federalist-proxy.app.cloud.gov/preview/onrr/doi-extractives-data/3016-prod-data-update/explore/#production)

Changes proposed in this pull request:

- Updated Production data
-
-

### CODE REVIEW

**Comment by jennmalcolm** [comment]
@master12, do you know why there isn't any coal data for 2017?

**Comment by master12** [comment]
it was the one thing that wasnt updated from the api call, but let me check on the api. maybe something changed.

**Comment by master12** [comment]
I cant find any coal data for 2017 on their website. The last report i found was dated Nov 2017 and had only 2016 data? Can you find any? @jennmalcolm

**Comment by jennmalcolm** [comment]
@master12 I did find the 2017 production data on their quarterly production report for coal (https://www.eia.gov/coal/production/quarterly/), but I'm guessing that where the API is coming from is the aggregate coal mine production (https://www.eia.gov/coal/data/browser/), which they only have through 2016. On that page, it does say that the data are usually available in the fall following the data collection year, so I guess we might have a couple months to wait. We might as well update the other products and run again once the coal data gets finalized. Thanks.

File: _data/years.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ self_employment: [2007, 2016]
 
 disbursements: 2017
 
-all_production: [2007, 2016]
+all_production: [2007, 2017]
```
Comment (brentryanjohnson) [inline _data/years.yml]: @master12 Should the range be `2008, 2017` as it is for federal production?

**Pull request merged**